core/oracles/adaptors/ethereum.go

```diff
@@ -0,0 +1,17 @@
+package adaptors
+
+import (
+	"crypto"
+
+	"code.vegaprotocol.io/vega/core/oracles"
+)
+
+type EthereumAdaptor struct{}
+
+func NewEthereumAdaptor() *EthereumAdaptor {
+	return &EthereumAdaptor{}
+}
+
+f//unc (a *EthereumAdaptor) Normalise(data []byte, _ ...crypto.PublicKey) (*oracles.OracleData, error) {
```

Comment:
looks a bit borked :)

Reply:
Corrected thanks, not sure of that part is to stay anyway